scripts/legislation/africa/eritrea_english.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+function remove_all_text_before_first_header {
+  sed -n '/^CHAPTER I/,$p'
+}
+
+function add_square_brackets_to_list_numbers {
+    sed -E 's/(\()([0-9]+)(\))/[\2]/g' 
+}
+
+function append_pipe_to_article_title {
+  sed -E 's/Article: 37/Article 37/' | \
+  sed -E 's/^(Article .*)$/\1|/'
+}
+
+function move_article_titles_above_article_bodies {
+  sed -E 's/^(\([0-9]+\)) ([^|]+)\|/\2\n\1/'
+}
+
+function amend_errors_in_articles {
+    sed -E 's/\(([a-z])\)/[\1]/g' | \
+    
+    #Article 1
+    amend_error_in_article 1 'No' 'No\. 179\/2017"' | \
+    #Article 2
+    sed -E ':start;s/^(\(2\).*)\[[0-9]+\] /\1/;t start' | \
```

Comment:
Please leave the numbers in article 2 intact. I only wanted you to remove them in the Namibia case because of how that specific definitions article is structured.